models/model.js

```diff
@@ -34,3 +34,20 @@ exports.fetchArticles = () => {
         }
     });
 };
+
+exports.fetchComments = (id) => {
+    if (!/^[^A-z]+$/.test(id)) {
+        return Promise.reject({ status: 400, message: 'Bad request! Enter a valid ID' })
+    } else {
+        return db.query(`SELECT * FROM comments 
+        WHERE article_id = $1 
+        ORDER BY created_at DESC;`, [id])
+        .then((commentsData) => {
+            if (commentsData.rows.length === 0) {
+                return Promise.reject({ status: 404, message: 'Comments do not exist!' })
```

Comment:
This rejection will be wrongly triggered for articles which do not exist, such as article 9999999